pandas/tests/plotting/test_xlim_ylim_dtype.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import pandas as pd
+from datetime import datetime as dt
+import numpy as np
+
+"""targeting #GH40781"""
+
+# arbitrary df to test
+df = pd.DataFrame(
+    {
+        "A" : [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+        "B": [dt.now() for i in range(0, 10)]
+    }
+)
+
+
+def test_axis_lim_invalid():
+    global df
+    """
+    supplying ranges with str, datetime and
+    mixed int,str and float,str dtype to raise
+    ValueError. Valid dtypes are np.datetime64
+    int, and float.
+    """
```

Comment:
generally there is no need to write docstrings for tests, could you please remove?

Reply:
noted for future PRs and has been removed from current accordingly